pkg/scanners/quay/image_scan_test.go

```diff
@@ -92,5 +93,12 @@ func TestParseImageScan(t *testing.T) {
 
 	expectedScan, err := getImageScan()
 	assert.Nil(t, err)
-	assert.Equal(t, expectedScan, actualScan)
+	expectedFeatures := expectedScan.Data.Layer.Features
+	actualFeatures := actualScan.Data.Layer.Features
+	for i, e := range expectedFeatures {
+		protoassert.SlicesEqual(t, e.Executables, actualFeatures[i].Executables)
+		e.Executables = nil
+		actualFeatures[i].Executables = nil
+	}
+	assert.Equal(t, expectedScan, actualScan) //nolint:donotcompareproto proto field handled above and nilled
```

Comment:
Should this line be extracted in a dedicated function that goes to the required depths, splitting the field comparisons between `assert` and `protoassert` ?

Reply:
I thought about it but this will complicate the check so I go with `nil` solution and comparing the proto field manually